lntest/itest/lnd_test.go

```diff
@@ -5351,10 +5355,6 @@ type singleHopSendToRouteCase struct {
 	// routerrpc submits the request to the routerrpc subserver if true,
 	// otherwise submits to the main rpc server.
 	routerrpc bool
-
-	// mpp sets the MPP fields on the request if true, otherwise submits a
-	// regular payment.
-	mpp bool
```

Comment:
doesn't this mean we are no longer testing legacy payments? or do we need to do this because our invoices now require them?